rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -1495,9 +1495,9 @@ boolean restartTask(
                     UnknownTaskRestException, PermissionRestException;
 
     @PUT
-    @Path("jobs/{jobid}/tasks/{taskname}/restartTaskOnError")
+    @Path("jobs/{jobid}/tasks/{taskname}/restartInErrorTask")
     @Produces("application/json")
-    boolean restartTaskOnError(
+    boolean restartInErrorTask(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: org.ow2.proactive_grid_cloud_portal.scheduler.exception.NotConnectedRestException, org.ow2.proactive_grid_cloud_portal.scheduler.exception.UnknownJobRestException, org.ow2.proactive_grid_cloud_portal.scheduler.exception.UnknownTaskRestException, org.ow2.proactive_grid_cloud_portal.scheduler.exception.PermissionRestException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1160)